apps/web/pages/export.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  Button,
+  Card,
+  CircularProgress,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+} from '@usevenice/ui'
+
+import {
+  CheckCircleFilledIcon,
+  CodeIcon,
+  CopyTextIcon,
+  DownloadIcon,
+  SyncIcon,
+} from '@usevenice/ui/icons'
+
+import type {GetServerSideProps} from 'next'
+import Link from 'next/link'
+import {useEffect, useState} from 'react'
+import {PreviewResult, usePreviewQuery} from '../components/export'
+import {ExternalLink} from '../components/ExternalLink'
+import {PageHeader} from '../components/PageHeader'
+import {PageLayout} from '../components/PageLayout'
+
+const PREVIEW_LIMIT = 8
+const DOWNLOAD_LIMIT = 100
+
+interface ServerSideProps {
+  apiKey: string
+}
+
+export const getServerSideProps: GetServerSideProps<ServerSideProps> = async (
+  ctx,
+) => {
+  const {serverGetUser} = await import('../server')
+  const user = await serverGetUser(ctx)
+
+  const {z} = await import('@usevenice/util')
+  const apiKey = z.string().parse(user?.user_metadata?.['apiKey'])
```

Comment:
If `apiKey` is optional, we should have a fallback scenario for when it gets passed into the UI (and log as an error into Sentry). At some point we should have a way to regenerate it in account settings too

Alternatively a question for @tonyxiao, should it be optional? Also feels a little weird that even `user` is optional